docs/user/01-20-command-palette.md

```diff
@@ -0,0 +1,26 @@
+# Command Palette
+
+With the Kyma dashboard command palette feature, you can easily navigate through all your dashboard resources, such as workloads, deployments, etc.
+You can perform actions such as listing or accessing your namespaces, or preferences, or uploading YAML files.
+
+## Features
+
+You can use command palette to perform the following actions:
+
+- Access command palette, by pressing `cmd+k` on your keyboard or choosing **Quick Navigation**.
+
+  ![Quick navigation](assets/quick-navi.png)
+
+- Enter `?` to see all navigation options and the resources' aliases. For example, if you want to see your namespaces, it's enough to type `ns`.
```

Comment:
```suggestion
- To see all navigation options and the resources' aliases, enter `?`. For example, if you want to see your namespaces, it's enough to type `ns`.
```

Reply:
Generally, the SAP Style Guide recommends saying what to do before how (see [Logical Sequence](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/0e1ae688897b4bc185aa17002bae8ef4.html?state=DRAFT&q=logic#logical-sequence)), and the examples indicate the preference of infinitives over gerunds. However, I'm leaving it up to you whether to apply the rule here. If you decide to go for it, please have a look at all the steps in this doc. I don't want to spam you with multiple suggestions of the same kind.